src/components/layout/Navigation.tsx

```diff
@@ -121,6 +128,7 @@ const Navigator = (): JSX.Element | null => {
       renderRoot={renderRoot}
       buildMenuItemId={buildNavigationLink}
       useChildren={useChildren as any}
```

Comment:
I guess this was already from before, but is it possible to fix this `any` ? Or should we wait for the new version that hook anyway ?